packages-web/badge/src/Badge.xml

```diff
@@ -1,92 +1,66 @@
-        </property>
-        <property key="openPageAs" type="enumeration" defaultValue="content">
-            <caption>Open page as</caption>
-            <category>Events</category>
-            <description/>
-            <enumerationValues>
-                <enumerationValue key="content">Full page</enumerationValue>
-                <enumerationValue key="popup">Popup</enumerationValue>
-                <enumerationValue key="modal">Blocking popup</enumerationValue>
-            </enumerationValues>
-        </property>
-        <property key="microflow" type="microflow" required="false">
-            <caption>Microflow</caption>
-            <category>Events</category>
-            <description>The microflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
-        <property key="nanoflow" type="nanoflow" required="false">
-            <caption>Nanoflow</caption>
-            <category>Events</category>
-            <description>The nanoflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
+        <propertyGroup caption="General">
+            <propertyGroup caption="General">
+                <property key="type" type="enumeration" required="true" defaultValue="badge">
+                    <caption>Type</caption>
+                    <description>Render it as either a badge or a color label</description>
+                    <enumerationValues>
+                        <enumerationValue key="badge">Badge</enumerationValue>
+                        <enumerationValue key="label">Label</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="defaultValue" type="textTemplate" required="false">
+                    <caption>Default value</caption>
+                    <description>The badge value shown when no value is provided via the attribute</description>
+                    <translations>
+                        <translation lang="en_US">Badge</translation>
+                        <translation lang="nl_NL">Badge</translation>
+                    </translations>
+                </property>
+                <property key="valueAttribute" type="attribute" required="false">
+                    <caption>Value attribute</caption>
+                    <description>The attribute that contains the badge value</description>
+                    <attributeTypes>
+                        <attributeType name="String" />
+                        <attributeType name="Enum" />
+                        <attributeType name="Integer" />
+                        <attributeType name="Decimal" />
+                        <attributeType name="Long" />
+                    </attributeTypes>
+                </property>
+                <property key="bootstrapStyle" type="enumeration" defaultValue="default">
+                    <caption>Badge style</caption>
```

Comment:
Let rethink if this property should be named `Badge style`...

Reply:
Why not create a design property out of it?

Reply:
Renamed it to brand style

Reply:
Like it for now